theme/forms.py

```diff
@@ -225,6 +226,12 @@ class Meta:
         model = User
         exclude = ['last_login', 'date_joined', 'password']
 
+    # redefining the Django username model field in order to change the invalid username message
+    # original message indicates that / is a valid character
+    username = forms.CharField(validators=[validators.RegexValidator(r'^[\w.@+-]+$',
```

Comment:
@pkdash It'd be better to call a Django API function or something to validate user name rather than copying its regular expression in Django code since that regex code could change from version to version (e.g., it has already been changed from version 9 to version 10).